Robinhood/Robinhood.py

```diff
@@ -188,117 +208,137 @@ def instruments(self, stock):
 
         return res['results']
 
-    def quote_data(self, stock=''):
-        """fetch stock quote
 
-        Args:
-            stock (str): stock ticker, prompt if blank
+    def quote_data(self, stock=''):
+        """
+            Fetch stock quote
 
-        Returns:
-            (:obj:`dict`): JSON contents from `quotes` endpoint
+            Args:
+                stock (str): stock ticker, prompt if blank
 
+            Returns:
+                (:obj:`dict`): JSON contents from `quotes` endpoint
         """
+
         url = None
+
         if stock.find(',') == -1:
             url = str(self.endpoints['quotes']) + str(stock) + "/"
         else:
             url = str(self.endpoints['quotes']) + "?symbols=" + str(stock)
+
         #Check for validity of symbol
         try:
             req = requests.get(url)
             req.raise_for_status()
             data = req.json()
+
```

Comment:
extra whitespace between `try`/`except`?

Reply:
Fixed --> I added it for additional readability, but I also know it's not part of the style-guide